### PR TITLE
Player Component

### PR DESCRIPTION
**Pull request by ipatate**

### CODE REVIEW

File: components/Player.vue [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    type: String,
+    required: true,
+  },
+});
+// never change
+const type = 'audio/mpeg';
+// audio tag
+const audioPlayerElement = ref<HTMLAudioElement | null>(null);
+
+const state: {
```
Comment (xlanex6) [inline components/Player.vue:60]: we could be `useState` to avoid Pinia  for the player

https://v3.nuxtjs.org/api/composables/use-state

**Review by xlanex6 — APPROVED** [review]

File: components/Player.vue (hunk 1 above)
Comment (xlanex6) [reply components/Player.vue:60]: and it will be sync over component

File: components/Player/TimelinePlayer.vue [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+.progress-bar {
+  @apply appearance-none border-none rounded-full h-1 m-0 p-0 absolute top-[0%] bg-purpleDs w-full focus:outline-none;
+}
+
+::-webkit-progress-bar {
+  @apply bg-purpleDs rounded-full h-1;
+}
+
+::-webkit-progress-value {
+  @apply bg-yellowDs rounded-full h-2 relative -top-0.5;
```
Comment (xlanex6) [inline components/Player/TimelinePlayer.vue:65]: do you have to prefix for webkit ? 
tailwind not make it by default ?

File: components/Player/index.vue [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import {calculateTotalValue} from '../../helpers/player';
+import ButtonPlayer from './ButtonPlayer.vue';
+import TimelinePlayer from './TimelinePlayer.vue';
+import TimerPlayer from './TimerPlayer.vue';
+import SpeedPlayer from './SpeedPlayer.vue';
```
Comment (xlanex6) [inline components/Player/index.vue:46]: Auto Import not working ?

File: config.ts [hunk 4]
```diff
@@ -10,6 +8,7 @@ export const baseInfos: BaseInfosType = {
     'Le podcast sur le code, le développement web et les outils modernes.',
   prefixAudio:
     'https://chtbl.com/track/79E812/double-slash.ams3.cdn.digitaloceanspaces.com',
+  prefixAudioDev: 'https://double-slash.ams3.cdn.digitaloceanspaces.com',
   // https://www.youtube.com/feeds/videos.xml?channel_id=
   channelID: 'UCp5DGBAX2XNJXeOVAo7bICQ',
```
Comment (xlanex6) [inline config.ts:13]: `youtubeChannelId` 
because tomorrow
twitch , castro , overCast ....erc

File: layouts/default.vue [hunk 5]
```diff
@@ -1,5 +1,12 @@
...
   <div class="min-h-screen bg-haiti">
     <slot />
+    <div class="w-1/2 bg-gray-200">
+      <LazyWrapperPlayer />
+    </div>
   </div>
 </template>
+
+<script setup>
+import LazyWrapperPlayer from '~~/components/Player/WrapperPlayer.vue';
```
Comment (xlanex6) [inline layouts/default.vue:11]: Auto imports ?

File: layouts/default.vue [hunk 6]
```diff
@@ -1,5 +1,12 @@
 <template>
   <div class="min-h-screen bg-haiti">
     <slot />
+    <div class="w-1/2 bg-gray-200">
```
Comment (xlanex6) [inline layouts/default.vue:4]: We should design it
....
later

File: helpers/player.ts [hunk 7]
```diff
@@ -0,0 +1,15 @@
+/**
```
Comment (xlanex6) [inline helpers/player.ts:1]: `/helpers` and `/plugins` could be merge into one compossable file with several function into
exemple 
`useTimeCalculate()`

File: nuxt.config.ts [hunk 8]
```diff
@@ -32,6 +32,7 @@ export default defineNuxtConfig({
     public: {
       ...baseInfos,
       podcastInfos,
+      isDev: process.env.NODE_ENV === 'development',
```
Comment (xlanex6) [inline nuxt.config.ts:35]: Nice

File: components/Player/TimelinePlayer.vue (hunk 2 above)
Comment (ipatate) [reply components/Player/TimelinePlayer.vue:65]: i don't think. it's very specific property. Not exist without prefix

File: components/Player/index.vue (hunk 3 above)
Comment (ipatate) [reply components/Player/index.vue:46]: No :)

File: config.ts (hunk 4 above)
Comment (ipatate) [reply config.ts:13]: ok, i change that.

File: layouts/default.vue (hunk 5 above)
Comment (ipatate) [reply layouts/default.vue:11]: No :)

File: layouts/default.vue (hunk 6 above)
Comment (ipatate) [reply layouts/default.vue:4]: yes, of course

File: nuxt.config.ts (hunk 8 above)
Comment (ipatate) [reply nuxt.config.ts:35]: 👍🏼

File: helpers/player.ts (hunk 7 above)
Comment (ipatate) [reply helpers/player.ts:1]: you can use composable in template ?

Comment (xlanex6) [reply helpers/player.ts:1]: Yea! Load in script but use in template 
Should try Without load in the script

Comment (ipatate) [reply helpers/player.ts]: ok i've added 2 composables. 
For auto import, i tried lots of solution, none works !
If it's ok for you, i can merge this PR.

**Pull request merged**